packages/clerk-js/src/core/clerk.ts

```diff
@@ -1165,3 +1163,5 @@ export default class Clerk implements ClerkInterface {
     }
   }
 }
+
+const clerkSyncedParam = '__clerk_synced';
```

Comment:
❓ isn't there a `constants` or something like that module to this?

Reply:
Good question, I'm not sure. Open to any suggestions of course :smile:.

Reply:
Do you mean [this one](https://github.com/clerkinc/javascript/blob/00b44167c0cc789ebf0bf25af33cfbbe4a310d52/packages/clerk-js/src/core/constants.ts)?

Reply:
☝️

Reply:
I mean [this one](https://github.com/clerkinc/javascript/blob/824727a7bf52e6c30569e6e59476e7a34fed630f/packages/clerk-js/src/utils/getClerkQueryParam.ts#L7)

Reply:
@gkats @panteliselef I think it should go where @anagstef suggested

Reply:
I see, I was not aware of this pattern. I pushed a commit that should fix this :)

Reply:
@gkats you can squash it into yours